SteamKit2/SteamKit2/Types/DepotManifest.cs

```diff
@@ -312,6 +370,117 @@ void ParseProtobufManifestMetadata(ContentManifestMetadata metadata)
             CreationTime = DateUtils.DateTimeFromUnixTime( metadata.creation_time );
             TotalUncompressedSize = metadata.cb_disk_original;
             TotalCompressedSize = metadata.cb_disk_compressed;
+            EncryptedCRC = metadata.crc_encrypted;
+        }
+
+        byte[]? Serialize()
+        {
+            DebugLog.Assert( Files != null, nameof( DepotManifest ), "Files was null when attempting to serialize manifest." );
+
+            var payload = new ContentManifestPayload();
+            var uniqueChunks = new List<byte[]>();
+
+            foreach ( var file in Files )
+            {
+                var protofile = new ContentManifestPayload.FileMapping();
+                protofile.filename = file.FileName.Replace( '/', '\\' );

```

Comment:
This was incorrectly replacing slashes if the filename was not decrypted.

Discovered while adding a roundtrip test: https://github.com/SteamRE/SteamKit/commit/7559904d5f5d9c90a5d6ca14fe3e0003b720dbdd